tests/orm/node/test_node.py

```diff
@@ -749,7 +748,10 @@ def test_store_from_cache(clear_database_before_test):  # pylint: disable=unused
         data.put_object_from_tree(tmpdir)
 
     data.store()
-    with enable_caching():
-        clone = data.clone().store()
+
+    clone = data.clone()
+    clone._store_from_cache(data, with_transaction=True)  # pylint: disable=protected-access
```

Comment:
It is a bit weird that we have a caching test for a `Data` node even though they are now explicitly not cacheable. Can we change this in a `CalculationNode`? Should be straightforward change and then you can revert to using `with enable_caching()`

Reply:
I did try that (see my comment above), but it becomes complicated trying to make sure that the `CalculationNode` fulfills all the requirements of the `is_valid_cache` check. I'd rather not rely on that specific behavior (as we may change the `is_valid_cache` check again).

Reply:
What if `Node`s remain cacheable by default, but this is simply disabled at the `Data` level?

Reply:
This test is not possible with `Node` because they can not be stored.